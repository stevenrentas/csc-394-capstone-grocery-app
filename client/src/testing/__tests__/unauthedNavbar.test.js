import React from 'react';
import {render, fireEvent, screen} from '@testing-library/react';
import AuthNavbar from '../../unauthed/navbar/UnauthedNavbar';

describe('Unauthorized Navbar Components <unauthed/navbar/UnauthedNavbar.js>', () =>{
    test('renders components and makes snapshot', () => {
        const {container} = render(<AuthNavbar/>);
        expect(container).toMatchSnapshot();
    });

    test('Should render "FOOD GPT" title', () => {
        render(<AuthNavbar />);
        expect(screen.getByText(/FOOD GPT/)).toBeInDocument;
    });
});