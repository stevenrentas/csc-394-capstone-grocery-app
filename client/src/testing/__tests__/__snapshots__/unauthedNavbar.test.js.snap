// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized Navbar Components <unauthed/navbar/UnauthedNavbar.js> renders components and makes snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-i9gxme"
  >
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-6t7t6l-MuiPaper-root-MuiAppBar-root"
      id="navBar"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <h1
          id="appName"
        >
          FOOD GPT
        </h1>
      </div>
    </header>
  </div>
</div>
`;
